api_server/src/http_service.rs

```diff
@@ -795,6 +795,11 @@ mod tests {
         );
     }
 
+    #[test]
+    fn test_log_received_api_request() {
```

Comment:
I am strongly against tests that are only for coverage. If you do want to add a test because coverage is decreased, please write something where the expected result of the function is checked.